src/main.js

```diff
@@ -22,7 +22,9 @@ module.exports = config => {
   )
   const repoGitDiffHelper = new RepoGitDiff(config, metadata)
 
-  const lines = repoGitDiffHelper.getDiff(config, metadata)
+  let filteredLines = repoGitDiffHelper.getFilteredDiff(config, metadata)
```

Comment:
**suggestion(blocking):** we should call the method without parameters (issue from the past)

```suggestion
  const filteredLines = repoGitDiffHelper.getFilteredDiff()
```

Reply:
Yeah, no idea how that snuck in there.